scripts/utils/environment.js

```diff
@@ -179,4 +222,25 @@ export function validateEnvironmentDependencies(
       valid(environmentVersions.cookiecutter);
     }
   }
+
+  if (dependencies.includes(EnvironmentDependency.CLI)) {
```

Comment:
Let's tidy up the nested if statements if we can. Suggest returning early once conditions aren't met.

```typescript
 if (!environmentVersions.cli) {
  printInvalidMessage("cli is not available in your system");
  return;
}

if (!environmentVersions?.cli?.registry) {
  printInvalidMessage("cli is not available in your system");
  return;
}

const cliVersionMessage = `CLI Version current: ${environmentVersions?.cli?.local?.version} latest: ${environmentVersions?.cli?.registry?.latest}`;


```